pkg/operator/controller/certificate-publisher/publish_ca.go

```diff
@@ -19,6 +20,9 @@ func (r *reconciler) ensureDefaultIngressCertConfigMap(caBundle string) error {
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      name.Name,
 			Namespace: name.Namespace,
+			Annotations: map[string]string{
+				annotations.OpenShiftComponent: "Networking / router",
+			},
```

Comment:
You'll need to update `updateRouterCAConfigMap` and `deleteRouterCAConfigMap` if you want the operator to add this annotation on upgrade.

Reply:
Updated `routerCAConfigMapsEqual` to return false when annotations are not set or do not match the desired. This will trigger `Update` call in `updateRouterCAConfigMap`